app/components/Utility/CryptoLinkFormatter.jsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import PropTypes from "prop-types";
+import QRCode from "qrcode.react";
+
+/**
+ *  Given an asset name, amount and label generates uri for its refilling
+ *
+ */
+
+class CryptoLinkFormatter extends React.Component {
+    static assetTemplates = {};
+
+    static propTypes = {
+        asset: PropTypes.string,
+        address: PropTypes.string,
+        amount: PropTypes.number,
+        message: PropTypes.string,
+        size: PropTypes.number
+    };
+
+    static defaultProps = {
```

Comment:
Please include all non-mandatory props in defaults

Reply:
size is the only required prop. If i provide default values to another props like message we would get non useful info in link, like next: ethereum:0xded7ad236aaf95465c47?message= 
or double check if it's not empty + "noneptiness" cond for each type :)

Reply:
This true, and maybe more personal preference.

Using "undefined" as a defined value is bad practice, IMO, but it's fine. undefined is no assigned value, whereas null is an assigned value.